### ISSUE TITLE
Add database fields to UI

### ISSUE THREAD

**Issue by cstevenson1** [leeensminger/DelDOT-NPDES-Field-Tool#381]
@KCI-Ablowers 

There are a number of fields that are present in the database but not viewable/editable in the field tool. We would like to add the following:

- WORK_ORDER_PROBLEM_CODES.PRIORITY to Work Order portion of UI for structures and conveyances 
![image](https://user-images.githubusercontent.com/16919898/208975187-9d5ed54b-99ab-4123-ae45-282319898c3a.png)
*** This is currently somehow auto-populated on the backend and would need to be disabled (potentially using look up table LU_WORK_ORDER_CODES ?)

- BMPS.CONTRACT_NUM to Inventory portion of UI for BMPs (issue #377)
![image](https://user-images.githubusercontent.com/16919898/208974719-465a9f38-abac-470d-9660-ff37fa53d254.png)

- BMP_INSPECTIONS.MAINT_AREA_WORK to Inspection portion of UI for BMPs
![image](https://user-images.githubusercontent.com/16919898/208974789-2baf5fe9-6255-48a2-a614-a5fd96ca3016.png)

-WQ_INVESTIGATIONS.NEAREST_STREAM to WQ Investigations portion of UI for structures and conveyances
![image](https://user-images.githubusercontent.com/16919898/208978121-10d5da14-2fd6-42f1-ac5e-f0aa3880781b.png)

There could be a couple others but these are the main ones. If possible, please include level of effort if there are additional ones.

**Comment by kanglu666** [comment]
Yes, for the work order problem code priority , it's automatically populated based on the look up table "LU_WORK_ORDER_CODES".  
 This will need to change the current work flow add a customer editable column in the UI and let user to manually set the priority value and save to the DB.

all others are adding existing fields to the UI.
